examples/README.md

```diff
@@ -46,7 +46,7 @@ Cloud](https://anaconda.org/jbednar/notebooks). To run these notebooks on your
 own system, start up a Jupyter notebook server:
 
 ```
-jupyter notebook --NotebookApp.iopub_data_rate_limit=100000000
+BOKEH_RESOURCES=inline jupyter notebook --NotebookApp.iopub_data_rate_limit=100000000
```

Comment:
I ran into this issue: https://github.com/bokeh/bokeh/issues/5351. It seems helpful to default to this to prevent future issues.

Perhaps we should just add an extra note mentioning how to resolve it if it occurs, while continuing to use the CDN by default.